### PR TITLE
Cut v2.19.0-rc.0

### PR DESCRIPTION
**Pull request by codesome**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+## 2.19.0-rc.0 / 2020-06-08
+
+* [FEATURE] Discovery: Added discovery support for Triton global zones. #7250
```
Comment (brian-brazil) [inline CHANGELOG.md:3]: enhancement

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+* [FEATURE] TSDB: It now accepts callbacks to control series creation if you use TSDB as a package. #7159
```
Comment (brian-brazil) [inline CHANGELOG.md:4]: This is an internal API which has no impact on the binary, it should not get mentioned.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,16 @@
+## 2.19.0-rc.0 / 2020-06-08
+
+* [FEATURE] Discovery: Added discovery support for Triton global zones. #7250
+* [FEATURE] TSDB: It now accepts callbacks to control series creation if you use TSDB as a package. #7159 
+* [FEATURE] Added time range parameters for label names and label values API. #7288
```
Comment (brian-brazil) [inline CHANGELOG.md:5]: This is minor change to an existing feature, enhancement

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,16 @@
...
+* [ENHANCEMENT] TODO. #7188
+* [ENHANCEMENT] TSDB: Memory-map full chunks of Head (in-memory) block from disk. This reduces memory footprint and makes restarts faster. #6679
```
Comment (brian-brazil) [inline CHANGELOG.md:7]: This is big enough to be a feature

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,16 @@
...
+* [ENHANCEMENT] Increased timeout for alertmanager send failures. #7228
```
Comment (brian-brazil) [inline CHANGELOG.md:8]: This is misleading, it sounds like it's the http request timeout we changed. This is about resend-relay being more tolerant of failures.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,16 @@
...
+* [ENHANCEMEMT] Better collision resistance while checking for duplicate labels. #7058
```
Comment (brian-brazil) [inline CHANGELOG.md:9]: Collisions were eliminated, and I'd say bugfix.

**Comment by brian-brazil** [comment]
#7307 and #7318 should both be mentioned as bugfixes.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,15 @@
+## 2.19.0-rc.0 / 2020-06-08
+
+* [FEATURE] TSDB: Memory-map full chunks of Head (in-memory) block from disk. This reduces memory footprint and makes restarts faster. #6679
+* [ENHANCEMENT] TODO. #7188
```
Comment (roidelapluie) [inline CHANGELOG.md:4]: ?

Comment (codesome) [reply CHANGELOG.md:4]: I am trying to figure out the exact text to put in there, still a draft :)

Comment (brian-brazil) [reply CHANGELOG.md:4]: Hmm, this looks like a cleanup change with no impact in which case we wouldn't list it.

Comment (codesome) [reply CHANGELOG.md:4]: That saves time

**Comment by codesome** [comment]
fuzzit failure is unrelated. I am planning to merge this (after approvals obviously) with that failure.

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,16 @@
+## 2.19.0-rc.0 / 2020-06-08
+
+* [FEATURE] TSDB: Memory-map full chunks of Head (in-memory) block from disk. This reduces memory footprint and makes restarts faster. #6679
+* [ENHANCEMENT] Discovery: Added discovery support for Triton global zones. #7250
+* [ENHANCEMENT] Increased alert resend delay to be more tolerant towards failures. #7228
+* [ENHANCEMENT] Remote Read: Added `prometheus_remote_storage_remote_read_queries_total` counter to count total number of remote read queries. #7328
+* [ENHANCEMEMT] Added time range parameters for label names and label values API. #7288
+* [ENHANCEMENT] TSDB: Reduced contention in isolation for high load. #7332 
+* [BUGFIX] Eliminated collision while checking for duplicate labels. #7058
```
Comment (roidelapluie) [inline CHANGELOG.md:9]:
```suggestion
* [BUGFIX] PromQL: Eliminated collision while checking for duplicate labels. #7058 
```

**Comment by brian-brazil** [comment]
:+1:

**Comment by codesome** [comment]
Benchmark looks good

**Comment by codesome** [comment]
There is slight increase in CPU usage, might be the additional m-map tasks for the querier. Not sure. But the increase does not look concerning.

**Pull request merged**